src/scrappers/source/proCyclingStats/raceStageResults.js

```
@@ -200,361 +215,377 @@ function climb(label) {
-          nestedDiv: "div", // Actual time value
-        },
-      };
+      rows.push(rowDetails);
+    },
+  );
 
-      const columns = Array.from(
-        tableElement.querySelectorAll(tableStructure.headers),
-      ).map((cell) => cell.innerText.trim());
-      // TODO: cell.getAttribute('data-code') ?? cell.innerText.trim())
+  notices.forEach((notice) => {
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)